neo/tools/compilers/fontcomp/fontcomp.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * fontcomp.h
+ *
+ *  Created on: 26 de gen. 2018
+ *      Author: biel
+ */
+
+#include <RapidXML>
```

Comment:
Something with the include doesnt seem to be right, doesnt work with xcode atleast.